modules/crawler/src/test/scala/ParserTest.scala

```diff
@@ -38,8 +38,8 @@ object ParserTest extends SimpleIOSuite:
       "10001492       Ojok, Patrick                                                UGA M             FI,LSI             2700  0   20 1932  0   20 1926  0   20 1974      "
     )
     lines
-      .traverse(parse)
-      .map(_.flatMap(_.flatMap(_.standard)))
+      .traverseFilter(parse)
+      .map(_.flatMap(_.standard))
```

Comment:
@lenguyenthanh both `flatMap` and `mapFilter` can be used here, which one expresses intention better?

Reply:
I guess `mapFilter` expresses better as in the ideal wold `flatMap` shouldn't do filter I believe, its just a scala thing.